packages/authentication/azure/package.json

```diff
@@ -30,11 +31,12 @@
   "homepage": "https://github.com/microsoft/kiota-typescript#readme",
   "dependencies": {
     "@azure/core-auth": "^1.5.0",
-    "@microsoft/kiota-abstractions": "^1.0.0-preview.50",
+    "@microsoft/kiota-abstractions": "*",
```

Comment:
what's the rationale behind switching to a star reference?

Reply:
Since we are using Lerna to manage the monorepo, in local development using the * allows using the latest changes in a package.

Reply:
Yes, the published version will use the latest version of abstractions. Is that going to be problematic?

Reply:
no, that's what I expected. Thanks for confirming. Resolving